source/_components/brottsplatskartan.markdown

```diff
@@ -2,15 +2,15 @@
 layout: page
 title: "Brottsplatskartan"
 description: "Instructions on how to integrate brottsplatskartan.se into Home Assistant."
-date: 2018-12-02 20:00
+date: 2019-05-11 22:30
 sidebar: true
 comments: false
 sharing: true
 footer: true
 ha_category:
   - Social
 logo: brottsplatskartan.png
-ha_release: 0.85
+ha_release: 0.94
```

Comment:
Same

Reply:
Thanks, changed it back to the old value.